gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoints-group/configuration/api-endpoints-group-configuration.component.html

```diff
@@ -0,0 +1,26 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card *ngIf="schemaForm">
+  <gv-schema-form-group
+    ngDefaultControl
+    [formControl]="groupConfigurationControl"
+    [schema]="schemaForm"
+    [attr.readonly]="groupConfigurationControl.enabled ? null : true"
+    (:gv-schema-form-group:error)="onConfigurationError($event.detail)"
+  ></gv-schema-form-group>
```

Comment:
You should not use this component. It's an old one.
You should use this one instead => `<gio-form-json-schema>`